module/mempool/entity_requests.go

```diff
@@ -0,0 +1,14 @@
+package mempool
+
+import "github.com/onflow/flow-go/engine"
+
+const DefaultEntityRequestCacheSize = 100_000
```

Comment:
This seems much too big to me. I think in the range of 1000-10000 would be better, if not even smaller.

Dropping entity requests generally is low-cost - they will be retried.

Reply:
https://github.com/onflow/flow-go/pull/3404/commits/cc2ed9962e43cca9e8994c1395587cf2c037e9b9